packages/@adobe/spectrum-css-temp/components/stepper/skin.css

```diff
@@ -245,3 +245,34 @@ governing permissions and limitations under the License.
     }
   }
 }
+@media (forced-colors: active) {
+  .spectrum-Stepper {
+    --spectrum-textfield-text-color-disabled: GrayText;
+    --spectrum-textfield-border-color-disabled: GrayText;
+    --spectrum-textfield-quiet-border-color-disabled: GrayText;
+    --spectrum-textfield-quiet-border-color-key-focus: Highlight;
+    --spectrum-textfield-border-color-key-focus: Highlight;
+    &:not(.spectrum-Stepper--isQuiet) {
+      &.is-focused, &:focus {
+        outline: 3px solid Highlight;
+      }    
+    }
+    &.spectrum-Stepper--isQuiet {
+      &.is-focused, &:focus {
+        border-width: 2px;
+      }
+      &.focus-ring, &:focus-ring {
```

Comment:
Is this meant to apply to both `:focus` and `:focus-ring`? With mouse focus, I am seeing a large focus ring on all four sides of the input, but with keyboard focus it's only the line underneath. The latter seems correct to me.

Reply:
large focus ring was from the textfield bug